critiquebrainz/db/users.py

```diff
@@ -0,0 +1,44 @@
+from hashlib import md5
+import sqlalchemy
+from critiquebrainz import db
+
+
+def avatar(user):
```

Comment:
What's a `user`? If you want to implement it like that, then the argument must be documented. Don't make people read the function to guess the type and structure you require.